exec/java-exec/src/main/java/org/apache/drill/exec/rpc/security/ClientAuthenticatorProvider.java

```diff
@@ -101,4 +101,12 @@ public void close() throws Exception {
     AutoCloseables.close(authFactories.values());
     authFactories.clear();
   }
+
+  /**
+   * By default Kerberos and Plain factories are provided by Drill.
+   */
+  @Override
+  public boolean isOnlyPlainConfigured() {
+    return false;
```

Comment:
removed the explicit check for PLAIN mechanism. Hence the method no longer exists.